### PR TITLE
Some notes on troubleshooting layer2 mode.

### PR DESCRIPTION
**Pull request by till**
**What this PR does / why we need it**:

I was debugging layer2 mode for too long and had little idea what I was doing.

I asked on Slack a few weeks ago and @danderson agreed that docs are lacking. I figured some notes/tips were in order. Also added a paragraph explaining how MetalLB layer2 works (high-level), as I read 10 or so blog posts that all seem to mis-understand that you are not supposed to bind IPs to interfaces and end up creating a SPOF.

Anyway, I am happy to rewrite everything or add to it — according to what is needed/required/preferred/liked.

**Special notes for your reviewer**:

Thank you for MetalLB! 🎉

### CODE REVIEW

**Comment by till** [comment]
I signed it!

File: website/content/configuration/troubleshooting.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
+---
+title: Troubleshooting
+weight: 20
+---
+
+Once you setup a loadbalancer on Kubernetes and it doesn't _seem_ to work right away, here are a few tips to troubleshoot.
+
+SSH into one or more of your nodes and use `arping` and `tcpdump` to verify the ARP requests pass through your network.
+
+Below assumes you setup a loadbalancer with an IP of `192.167.1.240` and then we show usage of the commands and successful
```
Comment (rbirkby) [inline website/content/configuration/troubleshooting.md]: 167 or 168?

File: website/content/configuration/_index.md [hunk 2]
```diff
@@ -23,6 +23,10 @@ to announce service IPs. Jump to:
 Layer 2 mode is the simplest to configure: in many cases, you don't
 need any protocol-specific configuration, only IP addresses.
 
+Layer 2 mode does not require the IPs to be bound to the network interfaces
+of your worker nodes. It works by _flooding_ your local network with ARP
```
Comment (danderson) [inline website/content/configuration/_index.md]: Flooding is not the right term here (sounds negative, like a DoS attack). How about:

It works by responding to ARP requests on your local network directly, to give the machine's MAC address to clients.

File: website/content/configuration/troubleshooting.md (hunk 1 above)
Comment (danderson) [reply website/content/configuration/troubleshooting.md]: Yeah should be 168 please.

File: website/content/configuration/troubleshooting.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+### speaker logs
+
+In addition to the above, make sure to watch the logs of MetalLB's speaker component for ARP requests and responses (using [kubetail](https://github.com/johanhaleby/kubetail)):
+
+```
+$ kubetail -l component=speaker -n metallb-system
+...
+```
+
+## Platform specifics
```
Comment (danderson) [inline website/content/configuration/troubleshooting.md]: This part is now included in a FAQ section on the website, so you can just delete it.

**Comment by till** [comment]
@danderson @rbirkby Made all requested changes, thanks for the feedback! :)

**Comment by till** [comment]
(I rebased on master before updating (hence the force-push). But let me know if I should squash my commits, etc..)

**Comment by till** [comment]
@danderson Just wondering if there's anything else I can do.

**Comment by danderson** [comment]
Give me more time to review stuff ;). Merged, thanks!

**Pull request merged**